### DISCUSSION TITLE
linux-arm and linux-x86 in latest releases

### DISCUSSION THREAD

**Discussion by tokuhirom** [async-profiler/async-profiler#578] [Q&A] [answered]
Until 2.0 release, there're linux-arm and linux-x86 binaries in github releases. After that, there's no linux-arm and linux-x86 binaries.

Is there a documentation or github issues about this policy change?

**Comment by apangin** [comment, marked as answer]
[async-profiler 2.5](https://github.com/jvm-profiling-tools/async-profiler/releases/tag/v2.5) brought support for two more platforms: macOS/ARM64 and Linux/PPC64LE. Maintaining releases for all available platforms is getting expensive, so I stopped providing binaries for the least popular ones and excluded these platforms from the full testing cycle (they can be still built from sources and pass smoke tests).

If needed, I can maintain the requested platforms on a commercial basis.

**Reply by tokuhirom** [reply]
Thanks. I got it!
